app/views/partners/_partner_row.html.erb

```diff
@@ -35,4 +35,7 @@
       &nbsp;
     <% end %>
   </td>
+  <td>
+    <%= print_button_to approve_application_partner_path(partner_row), { text: "Approve Partner", icon: "thumbs-o-up", type: "success", size: "xs" }%>
```

Comment:
We should probably have this in a conditional that checks !partner.approved? and that the current user is an admin?

Reply:
Agreed.